MetaMask/metamask-mobile
# feat(2794): account permission selection

## **Description**
This PR adds the ability to edit account permissions for in app browser dapps.





## **Related issues**

Contributes to solve issue: https://github.com/MetaMask/MetaMask-planning/issues/2796

## **Manual testing steps**

I have added screenshot below to see how it looks.

1. Go to the in-app brower on the mobile wallet, connect to a dapp like uniswap
2. once connected click to top right icon to open the bottom sheet and start editing
3. examples in screenshots below

## **Screenshots/Recordings**


| Add 1 account       | Remove 1 account    |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/21982551-8332-4596-89ed-50443ed86898"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/e6bd540b-bafa-4883-a80a-bc138fcabcd1"> |



| Connect all      | Disconnect all   |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/f2eaa78d-671c-4ea9-af0e-05a5a548631c"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/14c43919-bbc0-49d3-beb9-62452c65b899"> |



| Add many     | Remove many   |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/e0452e2a-8bb8-4741-8ec8-3642e63baeab"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/7639f3c2-c214-4b25-8e38-8b4c243efba5"> |


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
